src/sotoki/utils/imager.py

```diff
@@ -153,7 +153,8 @@ def get_s3_key_for(self, url: str) -> str:
 
     def get_digest_for(self, url: str) -> str:
         """Unique identifier of that url"""
-        return zlib.adler32(url.encode("UTF-8"))
+        return hashlib.md5(url.encode("UTF-8")).hexdigest()  #nosec
```

Comment:
:+1 ; mind adding the missing space?
```suggestion
        return hashlib.md5(url.encode("UTF-8")).hexdigest()  # nosec
```

Reply:
oops